packages/ts/hilla-file-router/src/vite-plugin.ts

```diff
@@ -87,5 +90,20 @@ export default function vitePluginFileSystemRouter({
       server.watcher.on('change', changeListener);
       server.watcher.on('unlink', changeListener);
     },
+    transform(code, id): Promise<TransformResult> | TransformResult {
+      if (id.startsWith(fileURLToPath(_viewsDir))) {
```

Comment:
Should maybe also ignore files starting with `_` inside the views directory?

Reply:
Right, good point

Reply:
Done